src/reset.css

```diff
@@ -67,11 +67,11 @@ select {
   font: inherit;
 }
 
-/* Remove _all_ animations and transitions for people that prefer not to see them */
+/* Remove transitions for people that prefer not to see them */
+/* Animations should be handled manually */
```

Comment:
This nuclear approach protects people with are negatively affected by motion. Often devs either aren't aware of this media query _or_ they forget to account for it. With that in mind this line should be removed.